gems/sorbet-runtime/test/types/props/serializable.rb

```diff
@@ -494,26 +494,275 @@ class CustomTypeStruct < T::Struct
     prop :hash_both, T::Hash[CustomType, CustomType], default: {}
   end
 
+  class CustomTypeWrapper
+    include T::Props::Serializable
+    prop :struct, CustomTypeStruct
+  end
+
   describe 'custom type' do
     it 'round trips as plain value' do
       assert_equal('foo', CustomTypeStruct.from_hash({'single' => 'foo'}).serialize['single'])
     end
 
+    it 'raises serialize errors for members with a custom subtype' do
```

Comment:
The body of this `it` makes sense, but I'm not sure I understand the description. What does `members with a custom subtype` mean here? (Same general 

Oh, hmm, maybe this means something like `raises serialize errors when the types of members of a custom subtype are wrong`?

Reply:
Maybe "raises serialization errors when dprops with a custom subtype store the wrong datatype"?

And so on for some of the (largely cut-and-pasted) errors below.

Reply:
Yeah, that sounds good!